### PR TITLE
Rename packages

### PR DESCRIPTION
**Pull request by mwcz**
#74

 - Rename "sync-service" to "api"
 - Rename "spa-manager" to "manager"
 - Rename "path-proxy" to "router"

### CODE REVIEW

File: packages/manager/package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  "name": "@spaship/manager",
   "version": "0.3.5",
   "description": "The SPAship UI.",
   "main": "index.js",
-  "homepage": "https://github.com/spaship/spa-manager",
+  "homepage": "https://github.com/spaship/manager",
   "repository": {
     "type": "git",
-    "url": "git@github.com:spaship/spa-manager.git"
+    "url": "git@github.com:spaship/manager.git"
```
[the remark this answers is not in the document]
Comment (kunyan) [reply packages/manager/package.json]: Good catch !

[the remark this answers is not in the document]
Comment (mwcz) [reply packages/manager/package.json]: Yeah, good catch!  5050e1f has the homepage and repository link fixes.

File: packages/manager/package.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@spaship/spa-manager",
+  "name": "@spaship/manager",
   "version": "0.3.5",
   "description": "The SPAship UI.",
   "main": "index.js",
-  "homepage": "https://github.com/spaship/spa-manager",
+  "homepage": "https://github.com/spaship/manager",
```
[the remark this answers is not in the document]
Comment (mwcz) [reply packages/manager/package.json]: Fixed in 5050e1f

File: packages/api/package.json [hunk 3]
```diff
@@ -1,19 +1,19 @@
 {
-  "name": "@spaship/sync-service",
+  "name": "@spaship/api",
   "version": "0.3.5",
   "description": "The file synchronization service for the SPAship platform.",
-  "homepage": "https://github.com/spaship/sync-service",
+  "homepage": "https://github.com/spaship/ync-service",
```
Comment (sayak-sarkar) [inline packages/api/package.json]:
```suggestion
  "homepage": "https://github.com/spaship/sync-service",
```

**Pull request merged**